application/deed/views.py

```diff
@@ -126,7 +126,7 @@ def create():
 
             deed.save()
 
-            url = request.base_url + str(deed.token)
+            url = "/deed/" + str(deed.token)
```

Comment:
this is no longer a URL, it is a path

http://www.matthewratzloff.com/blog/wp-content/uploads/2012/03/know_your_url.png